src/icp_canister_backend/src/lib.rs

```diff
@@ -28,6 +28,13 @@ thread_local! {
             Principal::anonymous()
         ).expect("Failed to initialize StableCell")
     );
+
+    static DEPOSIT_COUNTER: RefCell<StableCell<u64, Memory>> = RefCell::new(
```

Comment:
Keep logical order in the global variables too.
MEMORY_MANAGER goes first
Then TOKEN_ID
Then put Deposit Counter and Deposits, so they will be "grouped" together.